selendroid-standalone/src/main/java/io/selendroid/standalone/server/grid/SelfRegisteringRemote.java

```diff
@@ -136,7 +146,7 @@ private JSONObject getConfiguration() throws JSONException {
       configuration.put("proxy", "org.openqa.grid.selenium.proxy.DefaultRemoteProxy");
     }
     configuration.put("role", "node");
-    configuration.put("registerCycle", 5000);
+    configuration.put("registerCycle", config.getRegisterCycle());
```

Comment:
What does this value mean? You named the startup parameter the same, but the value now got changed from `5s` to `60s`..